src/main/provisioning/sling.txt

```diff
@@ -58,31 +58,31 @@
     org.apache.sling/org.apache.sling.jcr.contentloader/2.2.4
     org.apache.sling/org.apache.sling.jcr.resource/3.0.10
     org.apache.sling/org.apache.sling.models.api/1.3.6
-    org.apache.sling/org.apache.sling.models.impl/1.4.6
+    org.apache.sling/org.apache.sling.models.impl/1.4.8
     org.apache.sling/org.apache.sling.resourceresolver/1.6.0
     org.apache.sling/org.apache.sling.serviceusermapper/1.4.0
     org.apache.sling/org.apache.sling.serviceuser.webconsole/1.0.0
     org.apache.sling/org.apache.sling.servlets.get/2.1.30
     org.apache.sling/org.apache.sling.servlets.post/2.3.26
     org.apache.sling/org.apache.sling.servlets.resolver/2.4.22
-    org.apache.sling/org.apache.sling.xss/2.0.4
+    org.apache.sling/org.apache.sling.xss/2.0.6
     javax.mail/mail/1.5.0-b01
     org.apache.geronimo.bundles/jstl/1.2_1
     org.apache.felix/org.apache.felix.http.whiteboard/4.0.0
 
 [artifacts startLevel=5]
-    org.apache.sling/org.apache.sling.starter.content/1.0.0
+    gr8pi.portal/gr8pi-portal-content/0.1.1
```

Comment:
I also assume you want to leave the starter content and not add the gr8pi stuff :-)